samples/samples/connection_sample_test.py

```diff
@@ -0,0 +1,82 @@
+import uuid
+
+from google.cloud import spanner
+import pytest
+
+import connection_sample
+
+
+def unique_instance_id():
+    """ Creates a unique id for the database. """
+    return f"test-instance-{uuid.uuid4().hex[:10]}"
+
+
+def unique_database_id():
+    """ Creates a unique id for the database. """
+    return f"test-db-{uuid.uuid4().hex[:10]}"
+
+
+INSTANCE_ID = unique_instance_id()
+DATABASE_ID = unique_database_id()
+
+
+@pytest.fixture(scope="module")
+def spanner_instance():
+    spanner_client = spanner.Client()
+    instance_config = "{}/instanceConfigs/{}".format(
+        spanner_client.project_name, "regional-us-central1"
+    )
+    instance = spanner_client.instance(INSTANCE_ID, instance_config)
+    op = instance.create()
+    op.result(120)  # block until completion
+    yield instance
+    instance.delete()
+
+
+@pytest.fixture(scope="module")
+def database(spanner_instance):
+    """ Creates a temporary database that is removed after testing. """
```

Comment:
Excess whitespace at the comment start. If to add a docstrings, it makes sense to add docstrings for `spanner_instance()` as well